### PR TITLE
Update diff package

### LINKED ISSUE #340: Automated security alert: diff fix needed
image

### PR DESCRIPTION
**Pull request by PolinaShneider**
Closes #340
Updated project dependencies, particularly `@types/mocha`: ^5.2.5 -> ^5.2.7, using `diff`: < 3.5.0

### CODE REVIEW

**Comment by PolinaShneider** [comment]
@rodionos, have I correctly understood that I need to add this package directly to `devDependencies` of `server` in order to update version displayed in `package-lock.json`?

**Comment by annstriganova** [comment]
@PolinaShneider, try to investigate, which package uses `diff`, probably, we need to update some of existing dependencies.

**Comment by PolinaShneider** [comment]
@annstriganova figured that out and updated project dependencies

**Comment by annstriganova** [comment]
Synchronize `package.json` and `client/package.json` with `server/package.json`.

File: client/package.json [hunk 1]
```diff
@@ -19,15 +19,15 @@
...
+		"vscode": "^1.1.35",
+		"vscode-languageclient": "^5.2.1"
 	},
 	"devDependencies": {
-		"@types/lodash": "^4.14.123",
-		"@types/mocha": "^5.2.0",
-		"@types/node": "^10.9.4",
+		"@types/lodash": "^4.14.135",
+		"@types/mocha": "^5.2.7",
+		"@types/node": "^12.0.10",
```
Comment (annstriganova) [inline client/package.json]: No need to update, this version is not LTS.

Comment (PolinaShneider) [reply client/package.json]: This only refers to `lodash`?

Comment (PolinaShneider) [reply client/package.json]: A bit confused which packages I should update

**Pull request merged**